pyinfra/api/connectors/ssh.py

```diff
@@ -420,8 +420,25 @@ def put_file(
         temp_file = state.get_temp_filename(remote_filename)
         _put_file(host, filename_or_io, temp_file)
 
+        # Make sure our sudo/su user can access the file
+        if su_user:
+            command = StringCommand('setfacl -m u:{0}:r'.format(su_user), temp_file)
+        elif sudo_user:
+            command = StringCommand('setfacl -m u:{0}:r'.format(sudo_user), temp_file)
```

Comment:
```suggestion
            command = StringCommand('setfacl', '-m', 'u:{0}:r'.format(sudo_user), temp_file)
```

Minor nitpick on the style, makes no difference in output terms but consistent with `StringCommand` use!